.github/workflows/tests.yaml

```diff
@@ -145,6 +145,9 @@ jobs:
           provenance: false
           tags: |
             ghcr.io/${{ github.repository }}:${{ github.sha }}
+          labels: |
+            git.repository=${{ github.repository }}
+            git.commit-sha=${{ github.sha }}
```

Comment:
Do we need _these_ `git.repository` and `git.commit` labels? [i get that we want to have the values, the point is really about the labels and their name]

The "standard" for this kind of information is defined by OCI [1], would be `org.opencontainers.image.source`, and `org.opencontainers.image.revision` (plus `org.opencontainers.image.version` for releases).

Thes can be automatically computed with the [metadata action](https://github.com/docker/metadata-action), providing both `labels` and `annotations` (and tags, but this would require defining what kind of tagging schemes we want). I wanted to introduce this when doing the migration, but did not get the time to try and test...

[1] https://github.com/opencontainers/image-spec/blob/main/annotations.md

Reply:
Good to know, I think at the time I chatted with @tcarmet and we just went for a pragmatic solution, I don't think we considered following a standard here.

I don't see a reason not to follow the standard eventually, although I may want to merge this one as is if only to minimize changes required and ease integration testing for the quite urgent work on S3C-6474.

In order to implement the standard properly, would it make sense to go back to using a custom workflow in https://github.com/scality/workflows, to avoid repeating the changes in every repository where we want it?